platform/ios/src/MGLMapboxEvents.m

```diff
@@ -252,9 +256,13 @@ - (void)pauseOrResumeMetricsCollectionIfRequired {
         [self pauseMetricsCollection];
         return;
     }
+
+    if ([self isLowPowerModeEnabled]) {
+        [self pauseMetricsCollection];
```

Comment:
Do we resume once the charge reaches a suitable level and Low Power Mode automatically shuts off?